app/screens/home/channel_list/categories_list/header/plus_menu/item.tsx

```diff
@@ -16,24 +16,24 @@ const PlusMenuItem = ({pickerAction, onPress}: PlusMenuItemProps) => {
 
     const menuItems = {
         browseChannels: {
-            icon: 'globe',
+            leftIcon: 'globe',
```

Comment:
To avoid this to happen again in the future, can we set the type of `menuItems` to something like this?
`{[key: string]: Omit<React.ComponentProps<typeof SlideUpPanelItem>, 'onPress'>}`

Reply:
Done.